pkg/analysis/passes/virusscan/virusscan_test.go

```diff
@@ -18,6 +18,9 @@ func isClamAvInstalled() bool {
 	if err != nil {
 		return false
 	}
+	if os.Getenv("SKIP_CLAMAV") != "" {
```

Comment:
intentional change.

running tests when clamav is locally installed takes a considerable amount of time, this makes it much faster to run all tests locally except for these very specific ones.

Reply:
having clamav does not sound nice 😆